### PR TITLE
Design teardown

### PR DESCRIPTION
**Pull request by Israel-Laguan**
# Resume

We reviewed the site and compare it to the original, also check the stickler linter, and *came to the conclusion that we met the requirements*.

# Proposal

We want to have a review of the code to know if we need to make any changes. If no changes needed we want the project to be ACCEPTED so we can move to the next project with a happy smile :)

# Comments

Not having to make all details was refreshing so we can concentrate on layout and design. We learned a lot about positioning in this project. Also we had the opportunity to improve our GitHub flow.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,237 @@
...
+        <div class="footerContent__blockParent">
+          <div class="footerContent__block">Text</div>
+        </div>
+        <div class="footerContent__blockParent">
+          <div class="footerContent__text --smallTxt"></div>
+        </div>
+        <div class="footerContent__blockParent">
+            <div class="footerContent__text --smallTxt"></div>
+          </div>
+        <div class="footerContent__image">Image</div>
```
Comment (whiz25) [inline index.html]: wrap your footer content in ```<a>``` tags. they should be links. add some text like ```link-1``` so they can be visible.

**Review by whiz25 — CHANGES_REQUESTED** [review]
## Good job :+1: 
You have done awesome job with this project. It meets most requirements but we need to make a required change before it will be complete.
- [ ] wrap the footer links in ```<a>``` tags just like I suggested in the comment.

Make that change and submit for another review.

File: index.html (hunk 1 above)
Comment (Israel-Laguan) [reply index.html]: Thanks, we have added `<a>` tags and a word `links` in each line. Its that ok?
image

**Review by maelfosso — APPROVED** [review]
# Congratulations :clap:  The project is approved :smile: 

Good luck with the next project.

**Pull request merged**